spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -257,7 +254,15 @@ const ConfigSteps = () => {
 			}
 			setLoggedInUser(OAuthManager.getUserDetails().username);
 			setLoaderForLogin(false);
-			await getOrganizations();
+			setOrganizations([]);
+			if (status === true) {
```

Comment:
```suggestion
			if (status) {
```

Reply:
Sorry it won't work, if status is instanace of AxiosError, `if(status)` will evaluate to true. 
Generally speaking, explicit coding is better, so `xxx === true` will always be clear and less error prone.

Reply:
Hmm, actually it is shortcutted via 
```
if (status instanceof AxiosError) {
				setError(modifyError(status, {}, { onClearGitHubToken: clearGitHubToken }));
				return;
			}
```

So in this case will be fine. But still, I prefer to make it clear, instead of relying in some previous check and early return.